src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -3406,6 +3395,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                                                 </div>
                                             );
                                         }}
+                                        blurInputOnSelect={true}
```

Comment:
Somehow the search is not working after select one option, it's only working when you remove the focus of the selection box then click it again. Adding this to remove the focus after select one option.